forms-flow-web/src/services/UserService.js

```diff
@@ -127,7 +127,7 @@ const userLogout = () => {
 };
 
 const setApiBaseUrlToLocalStorage = () => {
-  localStorage.setItem("bpmApiUrl", BPM_BASE_URL);
+  localStorage.setItem("bpmApiUrl", BPM_BASE_URL_EXT);
```

Comment:
There was some implementations in client using this url in form directly. Check there won't be any breakage @krishnan-aot / @SnehaSuresh-aot